tests/unit/utils/test_juju_utils.py

```diff
@@ -351,9 +351,9 @@ async def test_get_current_model_from_juju(self):
 
     async def test_get_full_juju_status(self):
         with mock.patch("cou.utils.juju_utils._get_model") as get_model:
-            mymodel = AsyncMock()
+            mymodel = mock.AsyncMock()
```

Comment:
AsyncMock is already imported, why do we want to add `mock` import and use `mock.AsyncMock`?

Reply:
Not sure why, replacing it. Thanks